src/assets/js/search.js

```diff
@@ -98,13 +98,20 @@
   }
 
   const getExcerpt = (content, searchQuery) => {
+    const fullRegex = new RegExp(searchQuery, 'ig')
+    const fullSearchTerm = {
+      term: searchQuery,
+      regex: fullRegex,
+      matchIndex: content.search(fullRegex),
+    }
+
     const searchTerms = searchQuery.split(' ').filter(i => i).map(term => {
       const regex = new RegExp(term, 'ig')
 
       return { term, regex, matchIndex: content.search(regex) }
     })
 
-    const firstMatchedTerm = searchTerms.find(term => term.matchIndex !== -1)
+    const firstMatchedTerm = fullSearchTerm.matchIndex !== -1 ? fullSearchTerm : searchTerms.find(term => term.matchIndex !== -1)
```

Comment:
I wonder if it would be simpler to parse this if we simply added `fullSearchTerm` to the start of `searchTerms` using `.unshift`? Then the `firstMatchedTerm` line could stay simple and the hash building is a little DRYer. The result is basically the same though, so it's no big deal

```javascript
const searchTermStrings = searchQuery.split(' ').filter(i => i)

searchTermStrings.unshift(searchQuery)

const searchTerms = searchTermStrings.map(term => {
```